src/main/java/mekanism/common/tile/TileEntityLogisticalSorter.java

```diff
@@ -95,22 +100,25 @@ public void onUpdate() {
                 outer:
                 for (TransporterFilter filter : filters) {
                     for (StackSearcher search = new StackSearcher(back, facing.getOpposite()); search.getSlotCount() >= 0; ) {
-                        InvStack invStack = filter.getStackFromInventory(search);
+                        InvStack invStack = singleItem ? filter.getItemAmountFromInventory(search, 1)
```

Comment:
I think this would be clearer to put the ternary as the number param `singleItem ? 1 : 64`

Reply:
well, this was used because there were two methods - `getStackFromInventory` and getItemAmountFromInventory, only one was overriden in `TItemStackFilter`, so I wanted to call it only in one of the cases.

Will rewrite this to require only one `getStackFromInventory` method and make it cleaner.